src/Stratis.Bitcoin.Features.MemoryPool/Interfaces/IMempoolRule.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Stratis.Bitcoin.Features.MemoryPool.Interfaces
+{
+    /// <summary>
+    /// Rule to be checked before a transaction is added to the mempool.
+    /// </summary>
+    public interface IMempoolRule
```

Comment:
As we discussed is the name `Rule` correct here this is more like a `Validation` or a `Check` I wonder.

Reply:
Yeah, maybe it could be renamed in a future PR if necessary